test-data/unit/check-classes.test

```diff
@@ -2785,6 +2785,16 @@ def f(TA: Type[A]):
     reveal_type(TA)  # E: Revealed type is 'Type[__main__.A]'
     reveal_type(TA.x)  # E: Revealed type is 'builtins.int'
 
+[case testMetaclassSubclass]
```

Comment:
This test case doesn't seem relevant, since it doesn't actually define a class that uses a metaclass. I checked that I get the correct output on master.